app/assets/javascripts/canvas_shim/account_settings.js

```diff
@@ -10,33 +10,33 @@ function flipIcon(element, threshDisabled) {
 }
 
 $(window).on("load", function(event) {
-  var initialVal = $('#account_settings_score_threshold').val();
-  $('#edit_school_threshold_btn').click(function(e) {
+  $('#edit_passing_thresholds_btn').click(function(e) {
```

Comment:
This PR is awwwffulllly large.

Reply:
do you have some *constructive* input on how we might fix that?